core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/AutoTuner.scala

```diff
@@ -889,6 +897,10 @@ class AutoTuner(
     val missingRapidsJarsEntry = classPathComments("rapids.jars.missing")
     val multipleRapidsJarsEntry = classPathComments("rapids.jars.multiple")
 
+    if (platform.platformName == PlatformNames.EMR) {
```

Comment:
Can we add a comment before that block to explain why we skip EMR?
we can also consider making this as a function in the platform. By default it returns True and EMR returns False.

```
if (platfrom.requirePathRecommendations()) {
 recommendClassPathEntries()
}
```

Reply:
thanks for the input.  added bool in Platforms.scala in latest commit.